expr.go

```diff
@@ -2118,6 +2125,19 @@ func maxValue(f64s []float64, absent []bool) float64 {
 	return m
 }
 
+func minValue(f64s []float64, absent []bool) float64 {
```

Comment:
This is returning the maximum value in the array.  Is the name wrong or is the code wrong?